usaspending_api/api_contracts/contracts/v2/agency/toptier_code/budgetary_resources.md

```diff
@@ -15,51 +15,55 @@ Returns budgetary resources and obligations for the agency and fiscal year reque
                 "type": "number"
```

Comment:
For some reason I can only get the mock API to respond successfully when I delete lines 10-16.  I think `Request` type and `Schema` are unnecessary for GET requests (this may be a problem in other files too)